v4/Models/Survey.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+namespace v4\Models;
+
+use Illuminate\Database\Eloquent\Model;
+use Ushahidi\App\Repository\FormRepository;
+use Ushahidi\Core\Entity\Permission;
+use Ushahidi\Core\Tool\Permissions\InteractsWithFormPermissions;
+
+class Survey extends Model
+{
+    use InteractsWithFormPermissions;
+    protected $table = 'forms';
+    protected $with = ['stages'];
+    public $timestamps = FALSE;
+    /**
+     * The attributes that should be hidden for serialization.
+     * @note this should be changed so that we either use the fractal transformer
+     * OR a policy authorizer which is a more or less accepted method to do it
+     * (which uses the same $hidden type thing but it's much nicer obviously)
+     *
+     * @var array
+     */
+    protected $hidden = ['description'];
+    /**
+     * The attributes that should be mutated to dates.
+     * @var array
+    */
+    protected $dates = ['created', 'updated'];
+
+    /**
+    * The attributes that are mass assignable.
+    * @var array
+    */
+    protected $fillable = [
+        'parent_id',
+        'name',
+        'description',
+        'type',
+        'disabled',
+        'require_approval',
+        'everyone_can_create',
+        'color',
+        'hide_author',
+        'hide_time',
+        'hide_location',
+        'targeted_survey'
+    ];
+
+    /**
+     * The accessors to append to the model's array form.
+     *
+     * @var array
+     */
+    protected $appends = ['can_create'];
```

Comment:
this makes it so that calculated properties like can_create make it _cleanly_ into the json responses we send